terraform/development/dynamodb.tf

```diff
@@ -0,0 +1,24 @@
+resource "aws_dynamodb_table" "assetsinformationapi_dynamodb_table" {
```

Comment:
Are we sure these details are all correct? Do we know how data will be retrieved and whether or not we need a range key?
There is already a ticket just to setup dynamo db, so you could just comment these (i.e. all 3 environments) out until you do the db ticket.

Reply:
Makes sense, I'll remove them for now :)